example/lib/demos/image_picker_bottom_sheet_example.dart

```diff
@@ -23,6 +23,7 @@ class ImagePickerBottomSheetExample extends StatelessWidget {
                 : Text('$counterImage'),
             controller: ImagePickerGridController(AssetCounter()),
             context: context,
+            onGoToSettings: (_){},
```

Comment:
i think you should make it nullable, and have default behavior, instead of `onGoToSettings` everytime use the image_picker_bottom_sheet

Reply:
I agreed with Khai

Reply:
In fact on android we will always need permission dialog for this case, but this dialog might be different on every project. So I don't think it should be nullable because if there's no permission dialog, it won't be agreed by Google

Reply:
but its a callback, if you don't given `onGoToSettings`, its will be default behavior, and still have permission dialog, so i don't think it will effect to permission dialog

Reply:
```suggestion
            onGoToSettings: (_) {},
```

Sure, pls update formart for example

Reply:
@nqhhdev this PR was your suggestion. WDYT ? Nullable or not ?
If nullable I think we can remove this PR

Reply:
ok let's keep it nullable and move on